packages/openactive-integration-tests/test/features/access/access-code/implemented/manual-access-codes-test.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-unused-vars */
+const chakram = require('chakram');
+const chai = require('chai'); // The latest version for new features than chakram includes
+const { FeatureHelper } = require('../../../../helpers/feature-helper');
+const { GetMatch, C1, C2, B } = require('../../../../shared-behaviours');
+
+/* eslint-enable no-unused-vars */
+
+FeatureHelper.describeFeature(module, {
+  testCategory: 'access',
+  testFeature: 'access-code',
+  testFeatureImplemented: true,
+  testIdentifier: 'manual-access-codes',
+  testName: 'Successful booking with access codes.',
+  testDescription: 'Access codes returned for B request.',
+  // The primary opportunity criteria to use for the primary OrderItem under test
+  testOpportunityCriteria: 'TestOpportunityBookable',
+  // The secondary opportunity criteria to use for multiple OrderItem tests
+  controlOpportunityCriteria: 'TestOpportunityBookable',
+},
+function (configuration, orderItemCriteria, featureIsImplemented, logger, state, flow) {
+  beforeAll(async function () {
+    await state.fetchOpportunities(orderItemCriteria);
+
+    return chakram.wait();
+  });
+
+  afterAll(async function () {
+    await state.deleteOrder();
+    return chakram.wait();
+  });
+
+  describe('Get Opportunity Feed Items', function () {
+    (new GetMatch({
+      state, flow, logger, orderItemCriteria,
+    }))
+      .beforeSetup()
+      .successChecks()
+      .validationTests();
+  });
+
+  describe('C1', function () {
+    (new C1({
+      state, flow, logger,
+    }))
+      .beforeSetup()
+      .successChecks()
+      .validationTests();
+  });
+
+  describe('C2', function () {
+    (new C2({
+      state, flow, logger,
+    }))
+      .beforeSetup()
+      .successChecks()
+      .validationTests();
+  });
+
+  describe('B', function () {
+    (new B({
+      state, flow, logger,
+    }))
+      .beforeSetup()
+      .successChecks()
+      .validationTests();
+
+    it('Response should contain Access Code', function () {
+      chai.expect(state.bResponse.body.orderedItem[0].accessCode[0].description).to.equal('1234');
```

Comment:
This should be a schema check rather than a specific value check, as we're a real system could return any valid code.

E.g.
https://github.com/openactive/openactive-test-suite/blob/7e02bc9e17486af14d888267aae0505c15bcfa3d/packages/openactive-integration-tests/test/features/core/dataset-site/implemented/dataset-site-jsonld-valid-test.js#L30-L35